common/testAPIVersionHeader.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env node, mocha */
+var config
+var preparedRequest
+
+function testAPIVersionHeader (handler, verb, apiVersion) {
+  if (!apiVersion) {
+    apiVersion = '2.13'
+  }
+  if (apiVersion === '2.13') {
```

Comment:
Sorry, but I don't get it how this framework can be extended easily, maybe we could consider `gemver`?

Reply:
Considering OSB API is a back compatible API, we can write something like
```
if (semver.ge(apiVersion, '2.13') {
  run 2.13 cases here
}
if (semver.ge(apiVersion, '2.14') {
  run 2.14 new cases here
}
```
And `test.js` under the folder `2.14` also import the functions in under `common`.
Whatever, it should be another item in another PR. This PR wouldn't do that.